<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureMotionTouch</class>
 <widget class="QDialog" name="ConfigureMotionTouch">
  <property name="windowTitle">
   <string>设置 体感 / 触摸</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="motion_group_box">
     <property name="title">
      <string>体感</string>Motion
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="motion_provider_label">
          <property name="text">
           <string>体感提供程序：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="motion_provider"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="motion_sensitivity_label">
          <property name="text">
           <string>灵敏度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="motion_sensitivity">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="touch_group_box">
     <property name="title">
      <string>触摸</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="touch_provider_label">
          <property name="text">
           <string>触摸提供程序：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="touch_provider"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="touch_calibration_label">
          <property name="text">
           <string>校准：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="touch_calibration">
          <property name="text">
           <string>(100, 50) - (1800, 850)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="touch_calibration_config">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="touch_from_button_checkbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>使用按钮映射：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="touch_from_button_map"/>
        </item>
        <item>
         <widget class="QPushButton" name="touch_from_button_config_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="udp_config_group_box">
     <property name="title">
      <string>CemuhookUDP 设置</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="udp_help">
        <property name="text">
         <string>您可以使用任何与Cemuhook兼容的UDP输入源来提供体感和触摸输入。</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="udp_server_label">
          <property name="text">
           <string>服务器：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="udp_server">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="udp_port_label">
          <property name="text">
           <string>端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="udp_port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="udp_pad_index_label">
          <property name="text">
           <string>Pad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="udp_pad_index">
          <item>
           <property name="text">
            <string>Pad 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pad 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pad 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pad 4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="udp_learn_more">
          <property name="text">
           <string>学习更多</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="udp_test">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>167</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureMotionTouch</receiver>
   <slot>ApplyConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
